weave/integrations/exa/exa_sdk.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+
+import importlib
+from typing import TYPE_CHECKING, Any, Callable
+
+import weave
+from weave.integrations.patcher import MultiPatcher, NoOpPatcher, SymbolPatcher
+from weave.trace.autopatch import IntegrationSettings, OpSettings
+from weave.trace.weave_client import Call
+
+if TYPE_CHECKING:
+    pass
+
+
+_exa_patcher: MultiPatcher | None = None
+
+
+def exa_on_finish(call: Call, output: Any, exception: BaseException | None) -> None:
+    if getattr(output, "cost_dollars", None) and call.summary is not None:
+        # Create a usage entry for Exa with the cost information
+        exa_model_id = "exa"
+
+        # Get the cost as a float to ensure it's a numeric value
+        cost_value = float(output.cost_dollars.total)
```

Comment:
is cost not stored as a number?  That seems odd

Reply:
this doesn't work anyway...